dbhydra/dbhydra_core.py

```diff
@@ -533,7 +576,8 @@ def get_all_tables(self):
         return self.database.list_collection_names()
 
     def create_table(self, name):
-        return self.database[name]
+        table = self.database[name]
```

Comment:
self.database very unintuitive, it should probably be self.instance + it's questionable and needs to be checked